.dotnet/src/Custom/Embeddings/Embedding.cs

```diff
@@ -59,34 +59,15 @@ public partial class Embedding
     private EmbeddingObject Object { get; } = EmbeddingObject.Embedding;
 
     // CUSTOM: Added logic to handle additional custom properties.
-    /// <summary> Initializes a new instance of <see cref="OpenAI.Embeddings.Embedding"/>. </summary>
-    /// <param name="index"> The index of the embedding in the list of embeddings. </param>
-    /// <param name="embeddingProperty">
-    /// The embedding vector, which is a list of floats. The length of vector depends on the model as
-    /// listed in the [embedding guide](/docs/guides/embeddings).
-    /// </param>
-    /// <exception cref="ArgumentNullException"> <paramref name="embeddingProperty"/> is null. </exception>
-    internal Embedding(long index, BinaryData embeddingProperty)
```

Comment:
By deleting this constructor, it has moved back to the generated layer. But the constrcutor in the generated layer doesn't set the custom Vector property. I recommend restoring this constructor (and changing the long parameter to an int parameter) to ensure that all constructors work correctly. Alternatively, this constructor can be suppressed with CodeGenSuppress.

Reply:
Opted for the suppression route since it's not being used.